docs/08-add-component.md

```diff
@@ -0,0 +1,68 @@
+# Adding a component to the stack.
+
+## Vanilla
+
+---
+
+The vanilla version of the component, so it's a node package made of a style file and a javascript when needed, which exports a mixin of the component styles and the related javascript as a module.
+The name of the component is prefixed by _{system}-component_
+
+### Utilities packages
+
+There is a set of packages in the list of the vanilla ones which are providing utilities classes to be applied to the layouts when needed. Mind the fact that those utilities are not meant to be used in the component definition but only when defining layouts.  
+Their names are prefixed by _{system}-utility_
+
+**path:** src/systems/{system}/implementations/vanilla/packages/  
+**base_name:** ec-component-{component_name}  
+**files:** ec-component-{component_name}.scss, ec-component-{component_name}.js, package.json, README.md
+
+### Presets
+
+These are the presets collecting the different components, assembling them together and exporting those in a single package.
+At the moment the available presets are:
+
+**{system}-preset-dev** _(it imports all the available components)_  
+**{system}-preset-full** _(final css, js and images distributed in the dist folder with all the components included)_  
+**{system}-preset-website** _(final css, js and images distributed in the dist folder, recommended for projects using ECL)_
+
+## React components, pages and storybook
```

Comment:
again, in the title there are "components", "pages" and "storybook", but there are only two categories after.
We could add the "components" title first, with the same kind of information as the other types.